modules/imaging_browser/php/queryengine.class.inc

```diff
@@ -75,6 +75,15 @@ class QueryEngine extends \LORIS\Data\Query\SQLQueryEngine
                 new Cardinality(Cardinality::MANY),
                 $ScanType,
             );
+            $items[] = new QCDictionaryItem(
```

Comment:
Notes from meeting:
- This section adds the item to the data dictionary only
- QCDictionaryItem was selected because the data is closely coupled to the QCstatus data (files_qcstatus table) which already has a query defined in the `getFieldNameFromDict()` function below. It is a design choice not a mandatory one and the categorization of Dictionary items is ad-hoc for the moment.